ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,278 @@
+        curl -so /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles_mapping.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/elasticsearch/roles/roles_mapping.yml --max-time 300
+        curl -so /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/internal_users.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/elasticsearch/roles/internal_users.yml --max-time 300
+        rm /etc/elasticsearch/esnode-key.pem /etc/elasticsearch/esnode.pem /etc/elasticsearch/kirk-key.pem /etc/elasticsearch/kirk.pem /etc/elasticsearch/root-ca.pem -f
+        mkdir -p /etc/elasticsearch/certs $debug
+        cd /etc/elasticsearch/certs 
+        curl -so /etc/elasticsearch/certs/search-guard-tlstool-1.8.zip https://maven.search-guard.com/search-guard-tlstool/1.8/search-guard-tlstool-1.8.zip --max-time 300 
+        unzip search-guard-tlstool-1.8.zip -d searchguard 
+        curl -so /etc/elasticsearch/certs/searchguard/search-guard.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/searchguard/search-guard-aio.yml --max-time 300 
+        chmod +x searchguard/tools/sgtlstool.sh 
+        ./searchguard/tools/sgtlstool.sh -c ./searchguard/search-guard.yml -ca -crt -t /etc/elasticsearch/certs/ $debug 
+
+        if [ "$?" != 0 ]; then
+            echo "Error: certificates were not created"
+            exit 1
+        else
+            logger "Certificates created"
+        fi
+        rm /etc/elasticsearch/certs/client-certificates.readme /etc/elasticsearch/certs/elasticsearch_elasticsearch_config_snippet.yml search-guard-tlstool-1.7.zip -f $debug
+
+
+        # # Configure JVM options for Elasticsearch
+        jv=$(java -version 2>&1 | grep -o -m1 '1.8.0') || : 
+        if [ "${jv}" = "1.8.0" ]; then
+            ln -s /usr/lib/jvm/java-1.8.0/lib/tools.jar /usr/share/elasticsearch/lib/
+            echo "root hard nproc 4096" >> /etc/security/limits.conf
+            echo "root soft nproc 4096" >> /etc/security/limits.conf
+            echo "elasticsearch hard nproc 4096" >> /etc/security/limits.conf
+            echo "elasticsearch soft nproc 4096" >> /etc/security/limits.conf
+            echo "bootstrap.system_call_filter: false" >> /etc/elasticsearch/elasticsearch.yml
+        fi
+        # Start Elasticsearch
+        startService "elasticsearch"
+        echo "Initializing Elasticsearch..."
+        until $(curl -XGET https://localhost:9200/ -uadmin:admin -k --max-time 120 --silent --output /dev/null); do
+            echo -ne $char
+            sleep 10
+        done
+
+        cd /usr/share/elasticsearch/plugins/opendistro_security/tools/ $debug
+        ./securityadmin.sh -cd ../securityconfig/ -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key $debug
+
+        echo "Done"
```

Comment:
Use `logger` function.

Reply:
Done in 53208f3d9e390ee69e4f23799af5b3009001d9c2